src/Components/Shared/Button/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './button.module.css';
+
+const Button = ({ text, clickAction, type, disabled }) => {
+  let classNameStyle;
+  switch (type) {
+    case 'delete':
+      classNameStyle = styles.delete;
+      break;
+    case 'edit':
+      classNameStyle = styles.edit;
+      break;
+    case 'add':
+      classNameStyle = styles.add;
+      break;
+    case 'disabled':
+      classNameStyle = styles.disabled;
```

Comment:
In this case, the "type" prop of "disabled" would not be necessary as the "disabled" property itself already contains the necessary functionality.